src/taskEvents/script.js

```diff
@@ -0,0 +1,283 @@
+import {
+  getTaskLogs,
+  getUserData,
+  getSelfDetails,
+  createElement,
+  getData,
+  addLoader,
+  addErrorMessage,
+  removeLoader,
+} from './utils.js';
+
+import { createEventCard } from './logCard.js';
+
+const container = document.getElementById('taskEvents-container');
```

Comment:
please be consistent with the casing you use, what are we using here kebab case or a camel case?